gufe/network.py

```diff
@@ -98,7 +98,9 @@ def _defaults(self):
 
     def to_graphml(self) -> str:
         """ """
+        raise NotImplementedError
```

Comment:
@dwhswenson I thought we had this already, or was this a different Network object?

Reply:
`LigandNetwork`, not `AlchemicalNetwork`. We actually don't use `AlchemicalNetwork` yet.